### PR TITLE
feat: google analytics events

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

File: src/context/metrikaContext/metrikaContext.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import {Metrika, Pixel} from '../../models';
+import {GaInline, Metrika, Pixel} from '../../models';
 
 export interface MetrikaContextProps {
     metrika?: Metrika;
     pixel?: Pixel;
+    gaInline?: GaInline;
```
Comment (aapolkovsky) [inline src/context/metrikaContext/metrikaContext.tsx:7]: here and anywhere else gaInline -> ga or googleAnalytics

File: src/models/constructor-items/common.ts [hunk 2]
```diff
@@ -418,3 +421,29 @@ export interface BlockHeaderProps {
     title?: TitleProps | string;
     description?: string;
 }
+
+export declare type GaParameterValue = string | number | boolean;
```
Comment (aapolkovsky) [inline src/models/constructor-items/common.ts:425]: you don't need `declare` keyword

**Review by aapolkovsky — CHANGES_REQUESTED** [review]

File: src/context/metrikaContext/metrikaContext.tsx (hunk 1 above)
Comment (niktverd) [reply src/context/metrikaContext/metrikaContext.tsx]: ok

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gravity-ui/page-constructor",
-  "version": "1.12.0",
+  "version": "1.12.1",
```
Comment (gorgeousvlad) [inline package.json:3]: why do we update version manually in this pr?

**Comment by niktverd** [comment]
because of refactoring

**Pull request closed without merging**